client/src/Tickets/Tickets.jsx

```diff
@@ -129,25 +114,12 @@ const Tickets = () => {
     getPaginationRowModel: getPaginationRowModel(),
   });
 
-  const handleChange = (e) => {
-    const newData = { ...ticket };
-    newData[e.target.name] = e.target.value;
-    setTicket(newData);
-  };
-
   return (
     <main className="container">
       <div className="d-flex justify-content-between align-items-center mt-5">
-        <button type="button" className="btn btn-primary d-flex align-items-center" onClick={() => setToggleUserModal(true)}>
+        <Link className="btn btn-primary d-flex align-items-center" to="new">
           New <i className="bi bi-plus-lg" />
-        </button>
-        <AddTicketModel
-          update={updateTable}
-          data={ticket}
-          stateChange={handleChange}
-          toggleUserModal={toggleUserModal}
-          setToggleUserModal={setToggleUserModal}
-        />
+        </Link>
```

Comment:
Opening the modal becomes a simple route navigation that we can implement with a link...